operators/benchmark-receiver/run.py

```diff
@@ -0,0 +1,21 @@
+from typing import Any
+
+from interactem.core.logger import get_logger
+from interactem.core.models.messages import BytesMessage
+from interactem.operators.operator import operator
+
+logger = get_logger()
+
+@operator
+def receive_benchmark_frame(
+    inputs: BytesMessage | None, parameters: dict[str, Any]
+) -> None:
+
+    if inputs:
+        meta = inputs.header.meta if hasattr(inputs.header, 'meta') else {}
```

Comment:
In our receiver, we could have global variables that extract the metadata from each message, and create some kind of counter for each message size. When we finish receiving messages of a particular size, we could log a report that would indicate throughput, etc.